mmn/node_information.go

```diff
@@ -0,0 +1,125 @@
+package mmn
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os/exec"
+	"runtime"
+	"strings"
+
+	log "github.com/cihub/seelog"
+
+	"github.com/mysteriumnetwork/node/metadata"
+)
+
+type NodeInformation struct {
```

Comment:
We have almost all of this data as a quality metric: https://github.com/mysteriumnetwork/node/blob/master/core/quality/sender.go#L74
Maybe we should reuse it?

Reply:
What bothers we is WHAT data we collect. Ignas collects more detailed info. Such as Darwin version. `sw_vers -productVersion`. Which I suppose is needed. Maybe we can extend quality metrics with that info and then reuse that?

Reply:
My biggest worry here is that some providers probably will not want to report some information like OS details and MAC addresses because of privacy.

Reply:
We can hash it. MAC address is the only unique identifier we have for RPI nodes. We don't need it in plain text, but we do need a persistent property:)

Reply:
So, what we are going to do with it?

Reply:
I say hash it, or provide an ability to opt-out, e.g. `mymysterium.enabled=false`

Reply:
Solved